soccer/gameplay/skills/dribble.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, pos=None):
         self._threshold = 0.1  #default value matches the required accuracy for a placement command
         self._pos = pos
 
-        self._dribble_speed = 100
+        self._dribble_speed = 70
```

Comment:
This value should be relative to the maxDribbleSpeed, assuming the maxDribbleSpeed is what we use normally. (MaxDribbleSpeed can be tuned in the configs, and i'm expecting that we'll do that in the future).